db/version_set.cc

```diff
@@ -1218,6 +1218,9 @@ void Version::Get(const ReadOptions& read_options, const LookupKey& k,
       sample_file_read_inc(f->file_metadata);
     }
 
+    bool timer_enabled = get_perf_context()->per_level_perf_context_enabled &&
+                        GetPerfLevel() >= PerfLevel::kEnableTimeExceptForMutex;
```

Comment:
Reverting the two conditions feels slightly more performant in normal path to me.